tests/sample-server/src/main/kotlin/com/apollographql/apollo/sample/server/RootSubscription.kt

```diff
@@ -40,4 +45,26 @@ class RootSubscription : Subscription {
   fun operationError() = flow<String> {
     throw Exception("Woops")
   }.asPublisher()
+
+  @GraphQLDescription("Emits 'after' items and returns an error")
+  fun graphqlAccessError(after: Int = 1): Publisher<DataFetcherResult<Int?>> = flow {
+    repeat(after) {
+      emit(
+          DataFetcherResult.newResult<Int>()
+              .data(it)
+              .build()
+      )
+    }
+    emit(
+        DataFetcherResult.newResult<Int>()
+            .data(null)
+            .error(
+                GraphqlErrorException.newErrorException()
+                    .message("Woops")
+                    .build()
```

Comment:
For some reason, this error message doesn't make it through the other side. I'll add a comment for our future selves